docs/source/guides/network_node/staking_guide.rst

```diff
@@ -385,6 +387,23 @@ Wind down is *disabled* by default. To start winding down an existing stake:
     (nucypher)$ nucypher stake winddown --hw-wallet
 
 
+Snapshots
+*********
+
+Taking snapshots is *enabled* by default. This option gives ability to be involved in DAO. To stop taking snapshots:
+
+.. code:: bash
+
+    (nucypher)$ nucypher stake snapshots --disable
```

Comment:
@vzotova is there a reason someone would want to disable snapshots?

Reply:
Less gas spending on `mint` and `commitToNextPeriod`, iirc 25k gas less

Reply:
👍 - make sense. If you don't care about participating in the DAO, you save on some gas costs.